Makefile

```diff
@@ -0,0 +1,29 @@
+format:
+	go fmt ./...
+
+lint:
+	golangci-lint run
+
+build:
+	go build -o httpservergin ./httpServerGin/httpServerGin.go
+	go build -o httpserver ./httpServer/httpServer.go
+	chmod +x httpserver
+	chmod +x httpservergin
+
+
+docker-build:

```

Comment:
This command doesn't please fix it, fix your docker compose file
![Screenshot from 2024-07-16 16-24-32](https://github.com/user-attachments/assets/60cda32b-47f8-49b0-ac6c-e47b6689cfaa)